Update Connect documentation for 2.0 (#3887)
raspberrypi/documentation: documentation/asciidoc/services/connect/install.adoc

[[install-connect]]
== Install

> If Connect isn't already installed in your version of Raspberry Pi OS, open a Terminal window. Run the following command to update your system and packages:

Author: @spl237 suggested:

> Under Install, it might be worth mentioning that it can be installed from the Recommended Software GUI app, for those who don’t like terminal things. Only installs the full version, not lite.

Reviewer: Added!

$ sudo apt install rpi-connect

> ----

Reviewer: ```suggestion
----

You can also install Connect from the Recommended Software application.
```

After installation, use the `rpi-connect` command line interface to start Connect for your current user:

[source,console]
----
$ rpi-connect on
----

> Alternatively, if the Connect plugin is running in your menu bar, you can choose "Turn On Raspberry Pi Connect" from the dropdown menu:

Reviewer: ```suggestion
Alternatively, click the Connect icon in the menu bar to open a dropdown menu and select *Turn On Raspberry Pi Connect*:
```

$ sudo apt install rpi-connect-lite
----

After installation, use the `rpi-connect` command line interface to start Connect for your current user:

[source,console]
----
$ rpi-connect on
----

xref:connect.adoc#enable-remote-shell-at-all-times[Enable user-lingering] to make your device accessible even when your user account isn't logged in.

TIP: Lite commands use the same `rpi-connect` name as the full version of Connect. `rpi-connect-lite` is just a package name.

=== Manually start and stop Connect

To start Connect from the command line, run the following command:

[source,console]
----
$ rpi-connect on
----

To stop Connect, run the following command:

[source,console]
----

> $ rpi-connect on

Author: ```suggestion
$ rpi-connect off
```
